include/creadline.h

```diff
@@ -35,6 +35,8 @@
 #ifndef __CREADLINE_H__
 #define __CREADLINE_H__
 
+#include <stddef.h>
+
 /*! Override the maximum line size here (default: 64 MiB) */
 //#define CREADLINE_MAX_LINE_SIZE 10*1024*1024
```

Comment:
No idea why this is still in here but can be removed as well?

Reply:
Needed for type size_t